src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -45,6 +38,16 @@ $spacing: 20px;
     }
   }
 
+  &__icon {
+    flex-shrink: 0;
+    height: 180px;
+    width: 180px;
+    border-radius: 18px;
+    background-color: #000;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
```

Comment:
This should probably be using variables such as `$border-radius` (as per the styles on `img` removed in this PR), `$black` (or whatever colour we want to be used as the fallback/letterbox), etc.

Also probably want to include `background-position: center;` here as per https://github.com/sparkletown/sparkle/pull/1272#issuecomment-800768978 (and it would be good to see what a comparison screenshot looks like with `background-size: contain;` once this is added too)

Reply:
It looks very strange with `background-size: contain`
<img width="481" alt="Screen Shot 2021-03-25 at 6 23 18 PM" src="https://user-images.githubusercontent.com/51083763/112523968-3b55f800-8d97-11eb-909e-6340e55c0002.png">

But I agree with the `background-position: center;`, so I'll add that

Reply:
The Center with contain should centre the image so that the ‘letterboxing’ appears on both sides. In your screenshot above it looks like you may have only had contain but not centre set. We want both at the same time for it.